packages/core/src/connectors/injected.ts

```diff
@@ -20,6 +20,7 @@ export class InjectedConnector extends Connector<InjectedConnectorOptions> {
   }
 
   available() {
+    getStarknet()
     return globalThis['starknet'] !== undefined
```

Comment:
I think we could do this:
```suggestion
    return getStarknet() !== undefined
```

Reply:
Good point.

Reply:
I tested it with my code and it fails again.

Getting the a pollyfill from `getStarknet` will continue to fail when calling for the `ready` method, it calls `isPreauthorized` which is pollyfilled with a method which throw an Exception.

Reply:
Could you share the particular error it is throwing?